src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -924,7 +924,7 @@ def create_vm(cmd, vm_name, resource_group_name, image=None, size='Standard_DS1_
               enable_hotpatching=None, platform_fault_domain=None, security_type=None, enable_secure_boot=None,
               enable_vtpm=None, count=None, edge_zone=None, nic_delete_option=None, os_disk_delete_option=None,
               data_disk_delete_option=None, user_data=None, capacity_reservation_group=None, enable_hibernation=None,
-              v_cpus_available=None, v_cpus_per_core=None, accept_term=None, disable_integrity_monitoring=False,
+              v_cpus_available=None, v_cpus_per_core=None, accept_term=None, enable_integrity_monitoring=False,
```

Comment:
Please restore the definition of parameter `disable_integrity_monitoring` in the custom method, but do not add any logic for it anymore. This is because if this parameter is not defined in the custom method, it will be considered invalid, so the customer's automation scripts may still be break.

Reply:
Done